lredis/cqueues.lua

```diff
@@ -33,6 +33,35 @@ local function connect_tcp(host, port)
 	return new(socket)
 end
 
+local function connect(url)
+	local rest, auth,host,port,db
+	if url then
+		url = url:match("^redis://(.+)") or url
```

Comment:
I'd prefer we use proper uri parsing.
Have a look at https://github.com/daurnimator/lpeg_patterns#uri